downstream/modules/platform/proc-create-password-hashes.adoc

```diff
@@ -7,10 +7,15 @@
 +
 [literal, options="nowrap" subs="+quotes,attributes"]
 ----
-awx-manage shell_plus \
-from awx.main.utils import encrypt_value, get_encryption_key \
-postgres_secret = encrypt_value('$POSTGRES_PASS') \
-print(postgres_secret)
+# awx-manage shell_plus
+----
+. Then run the following from the python prompt
```

Comment:
Should a colon be included here? 

"Then run the following from the python prompt:"